modules/EnsEMBL/Web/Component/Shared.pm

```diff
@@ -320,32 +326,39 @@ sub transcript_table {
           $extras{$k} = $cell;
         }
       }
+
+      # Flag order: is_canonical, MANE_select, MANE_plus_clinical, gencode_basic, appris, TSL, CDS_start_NF, CDS_end_NF
+      my $refseq_url;
       if ($trans_attribs->{$tsi}) {
-        if (my $incomplete = $self->get_CDS_text($trans_attribs->{$tsi})) {
-          push @flags, $incomplete;
+        if ($trans_attribs->{$tsi}{'is_canonical'}) {
+          push @flags, helptip("Canonical", get_glossary_entry($hub, "Canonical transcript"));
+        }
+
+        foreach my $MANE_attrib_code (keys %MANE_attrib_codes) {
+          if (my $mane_attrib = $trans_attribs->{$tsi}{$MANE_attrib_code}) {
+            my ($mane_name, $refseq_id) = @{$mane_attrib};
+            $refseq_url  = $hub->get_ExtURL_link($refseq_id, 'REFSEQ_MRNA', $refseq_id);
+            push @flags, helptip($mane_name, get_glossary_entry($hub, $MANE_attrib_codes{$MANE_attrib_code}));
+          }
+        }
+
+        if ($trans_attribs->{$tsi}{'gencode_basic'}) {
+          push @flags, helptip('GENCODE basic', $gencode_desc);
+        }
+
+        if ($trans_attribs->{$tsi}{'appris'}) {
+          my ($code, $key) = @{$trans_attribs->{$tsi}{'appris'}};
+          my $short_code = $code ? ' '.uc($code) : '';
+          push @flags, helptip("APPRIS $short_code","<p>APPRIS $short_code: ".get_glossary_entry($hub, "APPRIS$short_code")."</p><p>".get_glossary_entry($hub, 'APPRIS')."</p>");
         }
+
         if ($trans_attribs->{$tsi}{'TSL'}) {
           my $tsl = uc($trans_attribs->{$tsi}{'TSL'} =~ s/^tsl([^\s]+).*$/$1/gr);
           push @flags, helptip("TSL:$tsl", "<p>TSL $tsl: ".get_glossary_entry($hub, "TSL $tsl")."</p><p>".get_glossary_entry($hub, 'Transcript support level')."</p>");
-	}
-      }
-      if ($trans_gencode->{$tsi}) {
-        if ($trans_gencode->{$tsi}{'gencode_basic'}) {
-          push @flags, helptip('GENCODE basic', $gencode_desc);
-        }
-      }
-      if ($trans_attribs->{$tsi}{'appris'}) {
-        my ($code, $key) = @{$trans_attribs->{$tsi}{'appris'}};
-        my $short_code = $code ? ' '.uc($code) : '';
-         push @flags, helptip("APPRIS $short_code","<p>APPRIS $short_code: ".get_glossary_entry($hub, "APPRIS$short_code")."</p><p>".get_glossary_entry($hub, 'APPRIS')."</p>");
- 
-      }
-      my $refseq_url;
-      foreach my $MANE_attrib_code (keys %MANE_attrib_codes) {
-        if (my $mane_attrib = $trans_attribs->{$tsi}{$MANE_attrib_code}) {
-          my $refseq_id = $mane_attrib->value;
-          $refseq_url  = $hub->get_ExtURL_link($refseq_id, 'REFSEQ_MRNA', $refseq_id);
-          push @flags, helptip($mane_attrib->name, get_glossary_entry($hub, $MANE_attrib_codes{$MANE_attrib_code}));
+	      }
```

Comment:
Indentation looks a bit weird here - have you set your editor to convert tabs to spaces?

Reply:
True, it was a hidden tab there (otherwise it's two spaces per level throughout the file). Fixed now.